pyk/src/pyk/tests/test_kcfg.py

```diff
@@ -276,46 +335,47 @@ def test_pretty_print(self):
                                 (3, 7, 0), (7, 6),           # Go to previous terminal node not as loop
                                 (3, 11, 0), (11, 8)          # Covered
                                 ),
-            'covers': cover_dicts((8, 11))                   # Loops back
+            'covers': cover_dicts((8, 11)),                  # Loops back
         }
         cfg = KCFG.from_dict(d)
 
-        print(set(map(lambda node: node.id, cfg.reachable_nodes(nid(5), reverse=True, traverse_covers=True))))
+        def _short_id(i) -> str:
+            return cfg.short_id(node(i))
 
         # TODO: Why are all nodes (besides the target) frontiers?
         # TODO: Add a cover
```

Comment:
If a node is interior, but not `expanded` or `covered`, it will be `frontier`.

You need to add `expanded` to nodes that have already had their successors computed.